engine/src/main/java/org/terasology/world/block/BlockRegion.java

```diff
@@ -518,6 +519,43 @@ public boolean containsPoint(Vector3fc point) {
         return this.aabb.containsPoint(point);
     }
 
+
+    /**
+     * Test whether the given aabb lies inside the {@link BlockRegion}
+     * @param region the region to test
+     * @return <code>true</code> iff the given value lies inside this {@link BlockRegion}; <code>false</code> otherwise
+     */
+    public boolean containsBlockRegion(BlockRegion region) {
+        return this.containsAABB(region.aabb);
+    }
+
+    /**
+     * Test whether the given {@link AABBi} lies inside the {@link BlockRegion}
+     * @param value the aabb to test
+     * @return <code>true</code> iff the given value lies inside this {@link BlockRegion}; <code>false</code> otherwise
+     */
+    public boolean containsAABB(AABBi value) {
+        return this.aabb.containsAABB(value);
+    }
+
+    /**
+     * Test whether the given {@link AABBf} lies inside the {@link BlockRegion}
+     * @param value the aabb to test
+     * @return <code>true</code> iff the given value lies inside this {@link BlockRegion}; <code>false</code> otherwise
+     */
+    public boolean containsAABB(AABBf value) {
+        return this.aabb.containsAABB(aabb);
+    }
+
+    /**
+     * Test whether the given {@link AABBd} lies inside the {@link BlockRegion}
+     * @param value the aabb to test
+     * @return <code>true</code> iff the given value lies inside this {@link BlockRegion}; <code>false</code> otherwise
+     */
+    public boolean containsAABB(AABBd value) {
+        return this.aabb.containsAABB(value);
+    }
+
```

Comment:
lets wait on this changes after this https://github.com/MovingBlocks/Terasology/pull/4321

Reply:
I'll remove these changes from this PR as they are not related to `BlockArea`.